client/settings/payment-settings/promotional-banner/bnpl-promotion-banner.js

```diff
@@ -0,0 +1,88 @@
+import { __ } from '@wordpress/i18n';
+import { React } from 'react';
+import styled from '@emotion/styled';
+import { ExternalLink } from '@wordpress/components';
+import CardBody from 'wcstripe/settings/card-body';
+import illustration from 'wcstripe/settings/payment-settings/promotional-banner/illustrations/bnpl.svg';
+import {
+	BannerIllustration,
+	ButtonsRow,
+	CardColumn,
+	CardInner,
+	DismissButton,
+} from 'wcstripe/settings/payment-settings/promotional-banner/banner-layout';
+
+const BannerIllustrationBNPL = styled( BannerIllustration )`
+	margin: 0 0 -40px 24px;
+`;
+
+const ButtonsRowBNPL = styled( ButtonsRow )`
+	margin-bottom: 0.7em;
+`;
+
+const IntroBNPL = styled.p`
+	line-height: 20px;
+`;
+
+const TitleBNPL = styled.h4`
+	margin-top: 0.6em !important;
+`;
+
+export const BNPLPromotionBanner = ( { setShowPromotionalBanner } ) => {
+	const handleBannerDismiss = () => {
+		setShowPromotionalBanner( false );
+	};
+
+	return (
+		<CardBody>
+			<CardInner>
+				<CardColumn>
+					<TitleBNPL>
+						{ __(
+							'Offer more ways to pay with Buy Now, Pay Later',
+							'woocommerce-gateway-stripe'
+						) }
+					</TitleBNPL>
+					<IntroBNPL data-testid="intro-bnpl">
+						{ __(
+							'Flexible pay-over-time options can boost revenue by up to 14%*.',
+							'woocommerce-gateway-stripe'
+						) }
+						<br />
+						{ __(
+							'Affirm and Klarna payments are auto-enabled with Stripe for eligible merchants.',
+							'woocommerce-gateway-stripe'
```

Comment:
QQ: is AfterPay not part of the promotion?

Reply:
@Mayisha confirmed that Afterpay is not part of the promotion (see pcUrMV-b0q-p2#comment-32318)